docs/nlu/choosing-a-pipeline.rst

```diff
@@ -142,19 +170,62 @@ these language models is available in the
 :ref:`SpacyFeaturizer` also provides word embeddings in many different languages (see :ref:`pretrained-word-vectors`),
 so you can use this as another alternative, depending on the language of your training data.
 
+Supervised Embeddings
+^^^^^^^^^^^^^^^^^^^^^
+
+If you don't use any pre-trained word embeddings inside your pipeline, you are not bound to a specific language
+and can train your model to be more domain specific. For example, in general English, the word "balance" is closely
+related to "symmetry", but very different to the word "cash". In a banking domain, "balance" and "cash" are closely
+related and you'd like your model to capture that.
+You should only use featurizers from the category :ref:`sparse featurizers <text-featurizers>`, such as
+:ref:`CountVectorsFeaturizer`, :ref:`RegexFeaturizer` or :ref:`LexicalSyntacticFeaturizer`, if you don't want to use
+pre-trained word embeddings.
+
+
 Entity Recognition / Intent Classification / Response Selectors
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 Depending on your data you may want to only perform intent classification, entity recognition or response selection.
 Or you might want to combine multiple of those tasks.
 We support several components for each of the tasks. All of them are listed in :ref:`components`.
 We recommend using :ref:`diet-classifier` for intent classification and entity recognition
 and :ref:`response-selector` for response selection.
 
-Comparing different pipelines for your data
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Rasa gives you the tools to compare the performance of both of these pipelines on your data directly,
-see :ref:`comparing-nlu-pipelines`.
+Multi-Intent Classification
+***************************
+
+You can use Rasa Open Source components to split intents into multiple labels. For example, you can predict
+multiple intents (``thank+goodbye``) or model hierarchical intent structure (``feedback_positive`` being more similar
+to ``feedback_negative`` than ``chitchat``).
+To do this, you need to use the :ref:`diet-classifier` in your pipeline.
+You'll also need to define these flags in whichever tokenizer you are using:
+
+    - ``intent_tokenization_flag``: Set it to ``True``, so that intent labels are tokenized.
+    - ``intent_split_symbol``: Set it to the delimiter string that splits the intent labels. Default ``_``.
```

Comment:
```suggestion
multiple intents (``thank+goodbye``) or model hierarchical intent structure (``feedback+positive`` being more similar
to ``feedback+negative`` than ``chitchat``).
To do this, you need to use the :ref:`diet-classifier` in your pipeline.
You'll also need to define these flags in whichever tokenizer you are using:

    - ``intent_tokenization_flag``: Set it to ``True``, so that intent labels are tokenized.
    - ``intent_split_symbol``: Set it to the delimiter string that splits the intent labels. In this case ``+``, default ``_``.
```
```suggestion
multiple intents (``thank+goodbye``) or model hierarchical intent structure (``feedback_positive`` being more similar
to ``feedback_negative`` than ``chitchat``).
To do this, you need to use the :ref:`diet-classifier` in your pipeline.
You'll also need to define these flags in whichever tokenizer you are using:

    - ``intent_tokenization_flag``: Set it to ``True``, so that intent labels are tokenized.
    - ``intent_split_symbol``: Set it to the delimiter string that splits the intent labels. Default ``_``.
```